crates/nu-command/tests/commands/length.rs

```diff
@@ -1,25 +1,15 @@
-use nu_test_support::{nu, pipeline};
+use nu_test_support::nu;
 
 #[test]
 fn length_columns_in_cal_table() {
-    let actual = nu!(
-        cwd: ".", pipeline(
-        r#"
-        cal | length -c
-        "#
-    ));
+    let actual = nu!("cal | length -c");
 
     assert_eq!(actual.out, "7");
 }
 
 #[test]
 fn length_columns_no_rows() {
-    let actual = nu!(
-        cwd: ".", pipeline(
-        r#"
-        echo [] | length -c
-        "#
-    ));
+    let actual = nu!("echo [] | length -c");
```

Comment:
we can remove the `echo` here :yum:

Reply:
(as i say in the top level review comment, this is not a requested change really as it's not related to the PR)

Reply:
As you said probable better scoped in a separate PR, as this touches only the test mechanics and not the actual test cases.

My slight hot take would probably be to keep some `echo` tests around as this will warn us if we decide to finally break it.